src/layer/BYOCLayer.ts

```diff
@@ -183,4 +183,16 @@ export class BYOCLayer extends AbstractSentinelHubV3Layer {
   protected getConvertEvalscriptBaseUrl(): string {
     return `${super.getConvertEvalscriptBaseUrl()}&byocCollectionId=${this.collectionId}`;
   }
+
+  public async getAvailableBands(reqConfig?: RequestConfiguration): Promise<void> {
+    const url = `https://services.sentinel-hub.com/api/v1/metadata/collection/CUSTOM/${this.collectionId}`;
```

Comment:
I would still check if `this.collectionId` is set and call `updateLayerFromServiceIfNeeded()` if it's not. Just to be on a safe side.

Reply:
True, it was added.

Reply:
:+1: Resolving.